jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHandler.java

```diff
@@ -94,7 +93,7 @@
  * <li>
  * Is the Response {@code Content-Length} header present, and does its
  * value meet the minimum gzip size requirements?
- * <br> (Default: 16 bytes. see {@link GzipHandler#DEFAULT_MIN_GZIP_SIZE})
+ * <br> (Default: 1024 bytes. see {@link GzipHandler#BREAK_EVEN_GZIP_SIZE})
```

Comment:
It seems like `DEFAULT_MIN_GZIP_SIZE` and `BREAK_EVEN_GZIP_SIZE` are independent of each other and you should have both. This javadoc doesn't make as much sense as its not linking to the default min gzip size here.